Include/internal/pycore_codecs.h

```diff
@@ -21,6 +21,17 @@ extern void _PyCodec_Fini(PyInterpreterState *interp);
 
 extern PyObject* _PyCodec_Lookup(const char *encoding);
 
+/*
+ * Un-register the error handling callback function error under the given
+ * name. Only non-built-in error handlers can be un-registered.
```

Comment:
Below you use "custom". Maybe it's better name than "non-built-in". I don't know.

Reply:
I reworded the docs. WDYT?